tee-worker/cli/src/trusted_base_cli/commands/get_storage.rs

```diff
@@ -0,0 +1,154 @@
+use crate::{command_utils::get_worker_api_direct, trusted_cli::TrustedCli, Cli};
+use codec::Decode;
+use ita_sgx_runtime::Runtime;
+use itc_rpc_client::direct_client::{DirectApi, DirectClient};
+use itp_rpc::{RpcRequest, RpcResponse, RpcReturnValue};
+use itp_types::DirectRequestStatus;
+use itp_utils::FromHexPrefixed;
+use log::{debug, error, warn};
+use my_node_runtime::Hash;
+use sp_application_crypto::scale_info::form::PortableForm;
+use std::sync::mpsc::channel;
+use subxt::{
+	self,
+	dynamic::{DecodedValue, DecodedValueThunk, Value},
+	ext::frame_metadata::StorageEntryType,
+	metadata::DecodeWithMetadata,
+	storage::{dynamic, utils, StorageAddress},
+	Metadata,
+};
+
+/// Usage:
+///    Plain Storage: ./integritee-cli trusted --mrenclave $mrenclave get-storage Parentchain Number
+///    Map Storage: ./integritee-cli trusted --mrenclave $mrenclave get-storage System Account 0xd43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d
+#[derive(Parser)]
+pub struct GetStorageCommand {
+	/// Pallet Name
+	pallet_name: String,
+
+	/// Storage Name
+	storage_name: String,
+
+	/// Stroage Key
+	keys: Vec<String>,
+}
+impl GetStorageCommand {
+	pub(crate) fn run(&self, cli: &Cli, trusted_args: &TrustedCli) {
+		let direct_api = get_worker_api_direct(cli);
+		let mrenclave = trusted_args.mrenclave.clone();
+		let storage_entry_keys: Vec<Value> = self
+			.keys
+			.iter()
+			.map(|v| {
+				let v = if v.starts_with("0x") {
+					v.strip_prefix("0x").unwrap().as_bytes()
+				} else {
+					v.as_bytes()
+				};
+				Value::from_bytes(hex::decode(v).unwrap())
+			})
+			.collect();
+		if let Some(v) = get_storage_value(
+			direct_api,
+			mrenclave,
+			self.pallet_name.as_str(),
+			self.storage_name.as_str(),
+			storage_entry_keys,
+		) {
+			println!("{}", v);
```

Comment:
I'm wondering what's the result of the non-number value? Hex-encoded strings? E.g. the identity in the IMT pallet

Reply:
It shoule be a JSON-String